jastrow.py

```diff
@@ -160,38 +208,47 @@ def __init__(self,nconfig,mol,basis=None):
         self.parameters={}
         self._nelec=np.sum(mol.nelec)
         self._mol=mol
-        self.parameters['bcoeff']=np.zeros(nexpand)
-        self.parameters['acoeff']=np.zeros(aexpand)
-        self._bvalues=np.zeros((nconfig,nexpand))
+        self.parameters['bcoeff']=np.zeros((nexpand, 3))
+        self.parameters['acoeff']=np.zeros((aexpand, 2))
+        self._bvalues=np.zeros((nconfig,nexpand, 3))
         self._configscurrent=np.zeros((nconfig,self._nelec,3))
         
         # First using gaussian, later change to obey cusp condition
-        self._avalues=np.zeros((nconfig,mol.natm,aexpand))
+        self._avalues=np.zeros((nconfig,mol.natm,aexpand, 2))
         
 
     def recompute(self,configs):
         """ """
         u=0.0
         self._configscurrent=configs.copy()
+        elec = self._mol.nelec
         #We will save the b sums over i,j in _bvalues
         
         #package the electron-electron distances into a 1d array
-        d=eedist(configs)
-        d=d.reshape((-1,3))
+        d1, d2, d3 = eedist(configs, elec[0], elec[1])
+        d1=d1.reshape((-1,3))
+        d2=d2.reshape((-1,3))
+        d3=d3.reshape((-1,3))
 
         # Package the electron-ion distances into a 1d array
-        di = eidist(configs, self._mol.atom_coords())
-        di = di.reshape((-1, 3))
+        di1, di2 = eidist(configs, self._mol.atom_coords(), elec[0], elec[1])
+        di1 = di1.reshape((-1, 3))
+        di2 = di2.reshape((-1, 3))
         
         for i,b in enumerate(self.b_basis):
-            self._bvalues[:,i]=np.sum(b.value(d).reshape( (configs.shape[0],-1) ),axis=1)
+            self._bvalues[:,i,0]=np.sum(b.value(d1).reshape( (configs.shape[0], -1) ),axis=1)
+            self._bvalues[:,i,1]=np.sum(b.value(d2).reshape( (configs.shape[0], -1) ),axis=1)
+            self._bvalues[:,i,2]=np.sum(b.value(d3).reshape( (configs.shape[0], -1) ),axis=1)
 
+        # Maybe bug is in here? Can't really tell since old Jastrow doesn't have spin
```

Comment:
Is there a bug here or not?

Reply:
There is not a bug. I deleted the comment.